ci/pipeline.yml

```diff
@@ -1,12 +1,5 @@
 ---
 jobs:
-- name: update-pulledpork-docker
-  plan:
-  - get: snort-release-source
```

Comment:
Will the hardened pulledpork image get built on every release of Snort? I assume not, which makes it slightly different than how this works, but I don't know if that matters

Reply:
I had to check again to be sure, but the common-pipelines method with look for any updates to this repo, and then create a new image if anything has changed, so it will basically operate the same way